src/source/source_cache.js

```diff
@@ -215,7 +215,7 @@ class SourceCache extends Evented {
         this._cache.reset();
 
         for (const i in this._tiles) {
-            this._reloadTile(i, 'reloading');
+            if (this._tiles[i] && this._tiles[i].state !== "errored") this._reloadTile(i, 'reloading');
```

Comment:
Is the first part of the condition necessary, since we're already iterating through the keys of `this._tiles`?